tests/validation/tests/v3_api/test_rke_cluster_provisioning.py

```diff
@@ -1076,7 +1076,8 @@ def register_host_after_delay(client, cluster, node_role, delay):
 
 
 def create_and_validate_custom_host(node_roles, random_cluster_name=False,
-                                    validate=True, version=K8S_VERSION):
+                                    validate=True, version=K8S_VERSION,
+                                    rke_config=rke_config):
```

Comment:
We are including rke_config in the list of variables for the method, how did it work before? 
Are there other calls to this method (in other files) which will need modification ?